tslint.json

```diff
@@ -25,9 +25,12 @@
         "newline-before-return": false,
         "no-backbone-get-set-outside-model": false,
         "no-implicit-dependencies": [
-            false, // TODO
+            true,
             [
-                "vscode"
+                "vscode",
+                "gulp",
+                "vscode-azureextensiondev",
+                "mocha"
```

Comment:
Suggest adding a `tslint.json` file to your test folder that allows all "dev" dependencies

Something like this: https://github.com/microsoft/vscode-azurefunctions/blob/master/test/tslint.json#L5

Reply:
Cool, thanks.